src/Contract/Wallet.purs

```diff
@@ -1,17 +1,19 @@
 -- | A module with Wallet-related functionality.
 module Contract.Wallet
   ( mkNamiWallet
+  , mkGeroWallet
   , module ContractAddress
   , module Wallet
   ) where
 
 import Contract.Monad (Contract)
 import Effect.Aff.Class (liftAff)
 import Wallet
-  ( NamiConnection
-  , NamiWallet
-  , Wallet(Nami)
+  ( Cip30Connection
```

Comment:
We are we exporting things like `Cip30Connection`? That should be considered internal

Reply:
```
  An export for Cip30Wallet requires the following to also be exported:

    Cip30Connection
```